data/campaigns/Dead_Water/scenarios/09_The_Mage.cfg

```diff
@@ -365,40 +442,202 @@
         [/if]
     [/event]
     # ***** END BATS SECTION********************END BATS SECTION*****
-
     [event]
         name=moveto
         [filter]
             side=1
-            x=8
-            y=5
+            [and]
+                race=bats
+                [or]
+                    variation=bat
+                [/or]
+            [/and]
+            x,y=8,5
```

Comment:
The attribute order looks odd here and the and tag again does nothing, right?

Reply:
The [and] tag makes a difference, because of the [or] tag.

This applies to (side=1 AND x,y=8,5 AND (race=bats OR variation=bat)).

Reply:
so, maybe add a comment to that effect, then?

Reply:
Right, so I'd just put x,y next to the other attributes. Not a blocker but it looks confusing otherwise IMO.